fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadLoadingTask.java

```diff
@@ -151,22 +151,22 @@ private void executeOnce() throws Exception {
          * here we use exec_mem_limit to directly override the load_mem_limit property.
          */
         curCoordinator.setLoadMemLimit(execMemLimit);
-        curCoordinator.setTimeout((int) (getLeftTimeMs() / 1000));
+
+        int timeoutS = (int) (getLeftTimeMs() / 1000);
```

Comment:
int 999 / 1000 = 0 in c++. It will have no chance to retry then even if it has 999ms left.

Reply:
PTAL